packages/vscode-extension/src/handlers.ts

```diff
@@ -1129,7 +1130,31 @@ async function openMarkdownHandler() {
 }
 
 export async function openReadMeHandler(args: any[]) {
-  if (workspace.workspaceFolders && workspace.workspaceFolders.length > 0) {
+  if (!(await isTeamsfx())) {
+    const createProject = {
+      title: StringResources.vsc.handlers.createProjectTitle,
+      run: async (): Promise<void> => {
+        createNewProjectHandler();
```

Comment:
This should be another triggerFrom  source for create-project telemetry

Reply:
there will be other PR to fix this, because the create project flow is a bit different for new walkthrough experience.